src/main/resources/l10n/JabRef_fr.properties

```diff
@@ -91,6 +91,8 @@ Automatically\ remove\ exact\ duplicates=Supprimer automatiquement les doublons
 
 AUX\ file\ import=Importation de fichier AUX
 
+author= Auteur
```

Comment:
Thin, `"author"` is not used in the code. This string is obsolete.

Please translate full sentences instead of part of sentences.